XPathParser/project.json

```diff
@@ -11,10 +11,11 @@
     "repository": {
       "type": "git",
       "url": "https://github.com/quamotion/XPathParser"
-    },
+    }
   },
 
   "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
I'm trying to move away from using NETStandard.Library and reference only the packages that are actually required. Is there a specific reason you had to take a dependency on NETSTandard.Library 1.6.0?